boards/Kconfig

```diff
@@ -2268,7 +2277,8 @@ config ARCH_BOARD
 	default "s32k146evb"               if ARCH_BOARD_S32K146EVB
 	default "s32k148evb"               if ARCH_BOARD_S32K148EVB
 	default "sabre-6quad"              if ARCH_BOARD_SABRE_6QUAD
-	default "sama5d2-xult"             if ARCH_BOARD_SAMA5D2_XULT
+    default "sama5d2-xult"             if ARCH_BOARD_SAMA5D2_XULT
+    default "giant-board"              if ARCH_BOARD_GIANT_BOARD
```

Comment:
Looks like spaces were used here instead of TABs.  Can you fix this?

Reply:
@patacongo Oops sorry, will fix. Have to change my editor settings for that! 🙂